### ISSUE TITLE
Matter accessories appear unresponsive in HomeKit (CON-490)

### ISSUE THREAD

**Issue by shun-dgut** [espressif/esp-matter#385]
Log output
```
E (2561939) chip[EM]: OnMessageReceived failed, err = 70
E (2574119) chip[EM]: OnMessageReceived failed, err = 70
E (2574179) chip[EM]: OnMessageReceived failed, err = 70
E (2599129) chip[EM]: Failed to Send CHIP MessageCounter:15364235 on exchange 53745i sendCount: 4 max retries: 4
E (2601539) chip[EM]: OnMessageReceived failed, err = 70
E (2601549) chip[EM]: OnMessageReceived failed, err = 70
E (2601819) chip[EM]: OnMessageReceived failed, err = 70
```
How do I locate and fix this bug

**Comment by dhrishi** [comment]
@shun-dgut This can be due to a variety of reasons. So, it is difficult to fix it for all the cases. The thing is, the device should still recover after this issue. Please let us know if that is not happening.

To give an example for this issue, in case of noisy networks where there are lot of retranmissions, it could happen that there are errors at the lwIP layer (like no memory) for sending the packets as the buffers get full. In such cases, if the upper layers handle retransmissions well then it would get resolved in any of the subsequent attempts

**Comment by dhrishi** [comment]
@shun-dgut If the device goes unresponsive, then please upload (and not paste) the device side log. In that case, I think this should be the issue and its fix: https://github.com/project-chip/connectedhomeip/pull/26215 .

We will soon update the connectedhomeip submodule to the next tag (when created) from the upstream with this fix.

**Comment by shun-dgut** [comment]
> @shun-dgut If the device goes unresponsive, then please upload (and not paste) the device side log. In that case, I think this should be the issue and its fix: [project-chip/connectedhomeip#26215](https://github.com/project-chip/connectedhomeip/pull/26215) .
> 
> We will soon update the connectedhomeip submodule to the next tag (when created) from the upstream with this fix.

Thanks，I am merging this for the test, and although the device will recover, it comes back after a while after recovery, and it will recover again.

My total stack is still with the remaining ones. I would like to open more logs to see.

**Comment by dhrishi** [comment]
@shun-dgut Please share your observations. We've now moved to connectedhomeip `v1.1-branch` which has this fix. Please close the issue if it works fine.
